enterprise/backup/src/main/java/org/neo4j/backup/BackupService.java

```diff
@@ -318,7 +319,8 @@ private BackupOutcome incrementalWithContext( String sourceHostNameOrIp, int sou
         DependencyResolver resolver = targetDb.getDependencyResolver();
 
         ProgressTxHandler handler = new ProgressTxHandler();
-        TransactionCommittingResponseUnpacker unpacker = new TransactionCommittingResponseUnpacker( resolver );
+        TransactionCommittingResponseUnpacker unpacker =
+                new TransactionCommittingResponseUnpacker( resolver, DEFAULT_BATCH_SIZE );
```

Comment:
I would assume it make sense instead of using static `DEFAULT_BATCH_SIZE` introduce property, so we can adapt batch size dynamically, ideally without restart with default value = `DEFAULT_BATCH_SIZE`